core/src/test/java/com/datastax/oss/driver/internal/core/util/concurrent/ReconnectionTest.java

```diff
@@ -295,7 +321,8 @@ public void should_stop_while_attempt_in_progress(boolean outcome) {
   @Test
   public void should_restart_after_stopped_while_attempt_in_progress() {
     // Given
-    Mockito.when(reconnectionSchedule.nextDelay()).thenReturn(Duration.ofNanos(1));
+    Mockito.when(reconnectionSchedule.nextDelay(Optional.empty()))
```

Comment:
I think that we are missing test case when `nextDelay()` is getting not `Optional.empty` as an argument

Reply:
There is one place where it's not an empty Optional: https://github.com/datastax/java-driver/pull/1172/files#diff-513c8e00357a995493fdfebc12eed677R65
But I think that is the only one.